### PR TITLE
 Fixed OnClick() and LongClick() and added ClockReset

### PR DESCRIPTION
**Pull request by KenNichols**

### CODE REVIEW

File: appinventor/components/src/com/google/appinventor/components/runtime/Chronometer.java [hunk 1]
```diff
@@ -277,35 +278,38 @@ public boolean onLongClick(View view)
      */
     @SimpleEvent
     public void Click() {
-        if (hasStarted)
```
Comment (Amerkashi) [inline appinventor/components/src/com/google/appinventor/components/runtime/Chronometer.java]: Why commenting out the logic? This means that Click won't really have any logic behind it

File: appinventor/components/src/com/google/appinventor/components/runtime/Chronometer.java [hunk 2]
```diff
@@ -277,35 +278,38 @@ public boolean onLongClick(View view)
...
+        //    Started();
+        //}
+        //hasStarted=!hasStarted;
         EventDispatcher.dispatchEvent(this, "Click");
     }
 
 
+    @SimpleEvent
+    public boolean LongClick() {
+        return EventDispatcher.dispatchEvent(this, "LongClick");
```
Comment (Amerkashi) [inline appinventor/components/src/com/google/appinventor/components/runtime/Chronometer.java]: Is this missing logic for handling the long-click?

File: appinventor/components/src/com/google/appinventor/components/runtime/Chronometer.java (hunk 1 above)
Comment (KenNichols) [reply appinventor/components/src/com/google/appinventor/components/runtime/Chronometer.java]: The Click Block will still function just like it does with the other Components

File: appinventor/components/src/com/google/appinventor/components/runtime/Chronometer.java (hunk 2 above)
Comment (KenNichols) [reply appinventor/components/src/com/google/appinventor/components/runtime/Chronometer.java]: The LongClick Block will still function just like it does with the other Components

**Comment by KenNichols** [comment]
editing code

**Pull request closed without merging**